pkg/imgpkg/bundle/images_lock.go

```diff
@@ -13,52 +13,94 @@ import (
 
 //go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 github.com/cppforlife/go-cli-ui/ui.UI
 
+type ImageRef struct {
+	lockconfig.ImageRef
+	IsBundle bool
+}
+type ImagesRef []ImageRef
```

Comment:
ImagesRef -> ImageRefs (plural of ImageRef, a single item in a collection). in other words, it's not a single Ref to bunch of images, it's multiple ImageRefs.

Reply:
Done